### PR TITLE
Rename chest to soko_box; add colors to "palettes", rename tokens that contain a tile and entity; 

### PR DESCRIPTION
**Pull request by ColtonPhillips**
Reason: mouth feel (i dont like saying chest), movability indicator (a ball is something you move), fidelity (a chest still sounds like a programming term, a ball is a gameplay item)

what you think? [https://giphy.com/gifs/ZKzFSESQpwuobp7XM8](what you think?)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -121,23 +121,23 @@ fn handle_move(prev_level: &types::Level, direction: types::Direction) -> types:
         }
     }
 
-    let mut chest_move = None;
+    let mut ball_move = None;
     if let Some((_, player_coords)) = player_move.clone() {
         for (index, entity) in level.entities.iter().enumerate() {
-            if let types::Entity::Chest(chest) = entity {
-                // if there is a chest where the player wants to move see if we can
+            if let types::Entity::Ball(ball) = entity {
```
Comment (lee-gauthier) [inline src/main.rs:127]: Lets call it SokoBox instead.

**Review by lee-gauthier — CHANGES_REQUESTED** [review]

File: src/types.rs [hunk 2]
```diff
@@ -94,9 +94,9 @@ impl Level {
 impl Tile {
     pub fn color(&self) -> Option<Color> {
         match self {
-            Tile::Wall => Some(get_color(TolColor::VibTeal)),
-            Tile::Goal => Some(get_color(TolColor::VibRed)),
-            Tile::Empty => Some(Color::Rgb(41, 19, 10)),
+            Tile::Wall => Some(get_color(TolColor::LigLightBlue)),
+            Tile::Goal => Some(get_color(TolColor::BriGrey)),
+            Tile::Empty => Some(Color::Rgb(222, 255, 255)),
```
[the remark this answers is not in the document]
Comment (ColtonPhillips) [reply src/types.rs]: OK done.

File: src/main.rs (hunk 1 above)
Comment (ColtonPhillips) [reply src/main.rs]: done. changed PlayerGoal to PlayerAndGoal in tokens

**Comment by ColtonPhillips** [comment]
Maybe what ive done to add my colors to TolColors is kinda dumb, but the function to get colors expects tol colors, and we might revamp this in the future anyway so solid for now i think

**Review by lee-gauthier — APPROVED** [review]
Looks good BB feel free to merge with main!

**Pull request merged**